food-chain/food_chain_test.rb

```diff
@@ -1,166 +1,11 @@
-    assert_equal expected, FoodChain.verse(7)
-  end
-
-  def test_horse
-    skip
-    expected = "" \
-      "I know an old lady who swallowed a horse.\n" \
-      "She's dead, of course!"
-    assert_equal expected, FoodChain.verse(8)
-  end
-
-  def test_multiple_verses
-    skip
-    expected = <<-SONG.strip
-I know an old lady who swallowed a fly.
-I don't know why she swallowed the fly. Perhaps she'll die.
-
-I know an old lady who swallowed a spider.
-It wriggled and jiggled and tickled inside her.
-She swallowed the spider to catch the fly.
-I don't know why she swallowed the fly. Perhaps she'll die.
-    SONG
-    assert_equal expected, FoodChain.verses(1, 2)
-  end
-
   def test_the_whole_song
-    skip
-    expected = <<-SONG.strip
-I know an old lady who swallowed a fly.
-I don't know why she swallowed the fly. Perhaps she'll die.
-
-I know an old lady who swallowed a spider.
-It wriggled and jiggled and tickled inside her.
-She swallowed the spider to catch the fly.
-I don't know why she swallowed the fly. Perhaps she'll die.
-
-I know an old lady who swallowed a bird.
-How absurd to swallow a bird!
-She swallowed the bird to catch the spider that wriggled and jiggled and tickled inside her.
-She swallowed the spider to catch the fly.
-I don't know why she swallowed the fly. Perhaps she'll die.
-
-I know an old lady who swallowed a cat.
-Imagine that, to swallow a cat!
-She swallowed the cat to catch the bird.
-She swallowed the bird to catch the spider that wriggled and jiggled and tickled inside her.
-She swallowed the spider to catch the fly.
-I don't know why she swallowed the fly. Perhaps she'll die.
-
-I know an old lady who swallowed a dog.
-What a hog, to swallow a dog!
-She swallowed the dog to catch the cat.
-She swallowed the cat to catch the bird.
-She swallowed the bird to catch the spider that wriggled and jiggled and tickled inside her.
-She swallowed the spider to catch the fly.
-I don't know why she swallowed the fly. Perhaps she'll die.
-
-I know an old lady who swallowed a goat.
-Just opened her throat and swallowed a goat!
-She swallowed the goat to catch the dog.
-She swallowed the dog to catch the cat.
-She swallowed the cat to catch the bird.
-She swallowed the bird to catch the spider that wriggled and jiggled and tickled inside her.
-She swallowed the spider to catch the fly.
-I don't know why she swallowed the fly. Perhaps she'll die.
-
-I know an old lady who swallowed a cow.
-I don't know how she swallowed a cow!
-She swallowed the cow to catch the goat.
-She swallowed the goat to catch the dog.
-She swallowed the dog to catch the cat.
-She swallowed the cat to catch the bird.
-She swallowed the bird to catch the spider that wriggled and jiggled and tickled inside her.
-She swallowed the spider to catch the fly.
-I don't know why she swallowed the fly. Perhaps she'll die.
-
-I know an old lady who swallowed a horse.
-She's dead, of course!
-    SONG
+    expected = IO.read("song.txt")
```

Comment:
Try using `File.expand_path('../song.txt', __FILE__)`. `make test` runs from the root of the repository, and would be looking for it relative to there, rather than relative to the test file.